### PR TITLE
Update/0.50 #3

### PR DESCRIPTION
**Pull request by danomatika**
Updates to master for 0.50 development cycle

- Closes: https://github.com/pure-data/pure-data/issues/235
- Closes: https://github.com/pure-data/pure-data/issues/850
- Closes: https://github.com/pure-data/pure-data/issues/878
- Closes: https://github.com/pure-data/pure-data/issues/883
- Closes: https://github.com/pure-data/pure-data/issues/895
- Closes: https://github.com/pure-data/pure-data/issues/896

### CODE REVIEW

**Comment by Spacechild1** [comment]
can I merge minor fixes into this branch or should we setup `update/0.51`?

**Comment by umlaeute** [comment]
depends on what you want to merge :-)
i guess *this* PR should only get (trivial) bugfixes for now

**Comment by umlaeute** [comment]
oh, and an `updates/0.51` should probably be based on `updates/0.50`

**Comment by danomatika** [comment]
We should probably rename this branch to 0.51... or better yet, merge it. :)

File: configure.ac [hunk 1]
```diff
@@ -562,6 +562,7 @@ AC_MSG_NOTICE([
     Installation prefix:  $prefix
 
     Compiler:             $CC
+    CPPFLAGS:             $CPPFLAGS
```
Comment (danomatika) [inline configure.ac]: Thanks. I was thinking the same thing...

**Comment by umlaeute** [comment]
@millerpuckette this PR is probably the lowest-hanging fruit in all the jungle of this repository.
i'm amazed it hasn't been merged yet

**Comment by danomatika** [comment]
We can always find more things...

**Comment by danomatika** [comment]
Latest commits fix a few zoom issues:

* incorrect margin and size values when opening canvas dialog while zoomed
* atom label position wrong while zoomed and moving with mouse
* triggerize placement doubled (wrong) while zoomed

**Pull request merged**